### PR TITLE
Adding new tasks and jobs for deploying hypervisor on target bare met…

### PR DESCRIPTION
**Pull request by rahmanmuhamad**
…al server via out of band controller using WSMAN protocol

### CODE REVIEW

**Comment by lanchongyizu** [comment]
@rahmanmuhamad Could you please help to add schema for task, and unit test for job/task/base-task?

File: lib/jobs/dell-wsman-os-create-job.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+
+     /**
+     *  Initialize basic configuration for the job
+     *
+     */
+    WsmanOsCreateJob.prototype._initJob = function () {
+        var self = this;
+        self.dellConfigs = configuration.get('dell');
+      if (!self.dellConfigs ||
```
Comment (nortonluo) [inline lib/jobs/dell-wsman-os-create-job.js]: Indent should be 4 space

**Comment by nortonluo** [comment]
@rahmanmuhamad the unit test does not require real hardware and real OS deployment. Only function test need do that thing. In unit test, we often mock/stub those thing. For example:  create a sandbox and reply a smi-service http REST request with designated JSON payloads {status:success} instead of do the real thing. This trick cheats the program to make the code under test "feel as run in real word".

**Comment by lanchongyizu** [comment]
depends on: https://github.com/RackHD/on-taskgraph/pull/345

**Pull request merged**